src/porepy/models/slightly_compressible_flow_model.py

```diff
@@ -0,0 +1,101 @@
+"""This module contains a simple extension of the incompressible flow by
+including a time derivative of the pressure and constant compressibility.
+"""
+
+import logging
+from typing import Dict, List, Optional
+
+import numpy as np
+
+import porepy as pp
+
+logger = logging.getLogger(__name__)
+
+
+class SlightlyCompressibleFlow(pp.models.incompressible_flow_model.IncompressibleFlow):
+    """This class extends the Incompressible flow model by including a
+    cummulative term expressed through pressure and a constant compressibility
+    coefficient. For a full documenation refer to the parent class.
+
+    The simulation starts at time t=0.
+
+    Overwritten methods include:
+        1. prepare_simulation:
+        Starting time set to 0, end time and timestep size optional paramters
+        2. _set_parameters: compressibility added
+
+    New methods:
+        1. _compressibility: constant compressibility per cell
+
+    Attributes:
+        end_time (float): Upper limit of considered time interval
+        time_step (float): time step size
+        time_index (int): number of time loops passed.
+        time (float): simulation time
+    """
+
+    def __init__(self, params: Optional[Dict] = None) -> None:
+        """
+        Parameters:
```

Comment:
The only parameter is params. Please update.

Reply:
Agree, it is done.